tests/pathfinding/test_capacity_updates.py

```diff
@@ -112,7 +94,7 @@ def test_pfs_rejects_capacity_update_with_wrong_chain_id(
 ):
     setup_channel(pathfinding_service_web3_mock)
 
-    message = get_updatepfs_message(
+    message = get_capacity_update_message(
```

Comment:
Good that you changed that, too! I already though we got all the updatepfs occurrences last time...